Misc/NEWS.d/next/Library/2019-03-14-15-21-01.bpo-31904.38fdkg.rst

```diff
@@ -0,0 +1 @@
+Remove crypt module support for VxWorks.
```

Comment:
Well, it's not like it was available previously. I propose to remove this NEWS entry and add a new one in the **Build** category:

```
Don't build the ``_crypt`` extension on VxWorks.
```